web/elm/src/SideBar/SideBar.elm

```diff
@@ -266,6 +275,16 @@ tooltip { hovered } =
                 , arrow = Just { size = Styles.tooltipArrowSize, color = Colors.tooltipBackground }
                 }
 
+        HoverState.Tooltip (SideBarInstanceGroup _ _ name) _ ->
```

Comment:
The tooltips for instance groups doesn't show up when you're in the Pipeline, Job, or Resource page. I suspect it's to do with with the fact `Pipeline.tooltip` is an nop, but not sure why the regular pipeline hovers work but not instance pipeline ones https://github.com/concourse/concourse/blob/94af3c750b0ccc28931054605a81fc0972e6c2e3/web/elm/src/Pipeline/Pipeline.elm#L436-L438

I can't figure out how to screenshot with cursor, but I'm hovering over `pipelin...` and there's no tooltip
![Screen Shot 2020-12-03 at 5 15 54 PM](https://user-images.githubusercontent.com/17577073/101095013-4c31f200-358b-11eb-83de-8a79a6965919.png)

However, hovering over regular pipelines work just fine
![Screen Shot 2020-12-03 at 5 15 21 PM](https://user-images.githubusercontent.com/17577073/101095028-4fc57900-358b-11eb-93da-0cd8a9f914a7.png)

Reply:
Good catch, fixed in 39c7c01e12fa04de5351f3240e9a447cb0b9b387